redis/src/cluster_routing.rs

```diff
@@ -719,7 +722,42 @@ mod tests {
             ),
         ] {
             assert_eq!(
-                RoutingInfo::for_routable(cmd),
+                RoutingInfo::for_routable(cmd, true),
+                expected,
+                "{}",
+                std::str::from_utf8(cmd.arg_idx(0).unwrap()).unwrap()
+            );
+        }
+    }
+
+    #[test]
+    fn test_routing_info_without_allowing_replicas() {
+        for (cmd, expected) in vec![
+            (
+                cmd("XINFO").arg("GROUPS").arg("foo"),
+                Some(RoutingInfo::SingleNode(
+                    SingleNodeRoutingInfo::SpecificNode(Route::new(slot(b"foo"), SlotAddr::Master)),
```

Comment:
what about test for SlotAddr::Replica, with allow_replicas == False? (or have I missed it?)